api/rds.proto

```diff
@@ -203,9 +203,15 @@ message ForwardAction {
   google.protobuf.BoolValue include_vh_rate_limits = 14;
 
   message HashPolicy {
-    string header_name = 1;
-    // Do we want to extend this for additional session affinity inputs?
-    // [V2-API-DIFF]
+    // [V2-API-DIFF] We expect additional hash policies in the future, e.g.
+    // cookie based, originating IP, etc.
+    oneof policy_specifier {
+      // The name of the request header that will be used to obtain the hash
+      // key. If the request header is not present, the load balancer will use a
+      // random number as the hash, effectively making the load balancing policy
+      // random.
+      string header_name = 1;
```

Comment:
e.g. StringMatch in https://github.com/istio/api/blob/master/proxy/v1/config/route_rule.proto#L278. It would be better to use native proto string match, but there is no such thing AFAIK.